lib/main.js

```diff
@@ -53,6 +54,13 @@ class Linter {
         this.registryEditors.createFromTextEditor(textEditor)
       }
     })
+    this.commands.onShouldShowLinterProviders(() => {
+      const linters = this.registryLinters.getLinters()
+      const textEditor = atom.workspace.getActiveTextEditor()
+      const textEditorScopes = textEditor.scopeDescriptorForBufferPosition(textEditor.getCursorBufferPosition()).getScopesArray().concat(['*'])
```

Comment:
Should this be moved to a helper function? This code is used in more than one place.